src/main/java/mate/academy/bookstore/repository/book/specification/AuthorSpecificationProvider.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.bookstore.repository.book.specification;
+
+import java.util.Arrays;
+import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.repository.SpecificationProvider;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AuthorSpecificationProvider implements SpecificationProvider<Book> {
+    public Specification<Book> getSpecification(String[] params) {
+        return (root, query, criteriaBuilder) ->
+                root.get("author").in(Arrays.stream(params).toArray());
```

Comment:
you are using literal `author` two times, so it will be better to move it to constant
same for other providers